src/main/java/org/systemexception/lunarlander/main/Main.java

```diff
@@ -0,0 +1,142 @@
+package org.systemexception.lunarlander.main;
+
+import org.jbox2d.common.Vec2;
+import org.jbox2d.dynamics.Body;
+import org.lwjgl.LWJGLException;
+import org.lwjgl.opengl.Display;
+import org.lwjgl.opengl.DisplayMode;
+import org.lwjgl.opengl.GL11;
+import org.newdawn.slick.Color;
+import org.newdawn.slick.TrueTypeFont;
+import org.newdawn.slick.opengl.TextureImpl;
+import org.newdawn.slick.util.ResourceLoader;
+import org.systemexception.lunarlander.constants.BodiesNames;
+import org.systemexception.lunarlander.physics.GameEngine;
+
+import java.awt.*;
+import java.io.InputStream;
+
+import static org.lwjgl.opengl.GL11.*;
+
+/**
+ * @author leo
+ * @date 29/12/15 14:22
+ */
+public class Main {
+
+	private static final String WINDOW_TITLE = "Physics in 2D!";
+	private static final int[] WINDOW_DIMENSIONS = {800, 600};
+
+	private final static double TWO_PI = 2 * Math.PI;
+
+	private TrueTypeFont font;
+	private static GameEngine gameEngine;
+
+	public static void main(String[] args) {
+
+		Main main = new Main();
+		gameEngine = new GameEngine();
+		main.start();
+	}
+
+	private void start() {
+		initGL();
+		initFonts();
+		gameEngine.setUpObjects();
+
+		while (!Display.isCloseRequested()) {
+			GL11.glClear(GL11.GL_COLOR_BUFFER_BIT);
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'GL11.GL_COLOR_BUFFER_BIT' due to existing import 'org.lwjgl.opengl.GL11'](https://www.codacy.com/app/leo_4/lunar-lander/file/1692129201/issues/source?bid=3040650&fileBranchId=3043231#l48)